article.html

```diff
@@ -10,12 +10,12 @@
 			<p>This will be the nav</p>
 		</header>
 		
-		<div class="wrapper">
+		<div class="article-wrapper">

```

Comment:
In this version, I changed the wrapper to "article-wrapper" because it was still the div for the body content on the screen, but it differed from all the other pages and needed a slightly different grid layout.